docs/griptape-framework/structures/tasks.md

```diff
@@ -103,7 +103,7 @@ agent = Agent()
 agent.add_task(
     ToolkitTask(
         "Load https://www.griptape.ai, summarize it, and store it in a file called griptape.txt", 
-        tools=[WebScraper(), FileManager(), TaskMemoryClient(off_prompt=False)]
+        tools=[WebScraper(off_prompt=True), FileManager(off_prompt=True), TaskMemoryClient(off_prompt=True)]
```

Comment:
Should be `False` for `TaskMemoryClient`.

Reply:
`FileManager` will read directly from Task Memory